docs/running-on-kubernetes.md

```diff
@@ -590,7 +590,8 @@ See the [configuration page](configuration.html) for information on Spark config
   <td><code>spark.kubernetes.driver.master</code></td>
   <td><code>https://kubernetes.default.svc</code></td>
   <td>
-    The internal Kubernetes master (API server) address to be used for driver to request executors.
+    The internal Kubernetes master (API server) address to be used for driver to request executors or
+    'local[*]' for driver-pod-only mode.
```

Comment:
Is local master URL format all supported? E.g, `local[K,F]`?

Reply:
Good point. I think we should probably leverage the same regex done in `SparkContextcreateTaskScheduler`, e.g., `LOCAL_N_REGEX`.

Reply:
Sure, we can do that.